Core/Session/LocalGrindingBotSessionHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace Core.Session
+{
+
+    // this is gonna save the bot session data locally atm
+    // there will be an AWS session handler later to upload the session data to AWS S3
+    // the idea is we will have two session data handlers working at the same time
+    public class LocalGrindingBotSessionHandler : IGrindingSessionHandler
+    {
+        private readonly string _historyPath;
+
+        public LocalGrindingBotSessionHandler(string historyPath)
+        {
+            _historyPath = historyPath;
+        }
+
+        public List<GrindingSession> Load()
+        {
+            // first time load
+            if (!Directory.Exists(_historyPath))
+            {
+                Directory.CreateDirectory(_historyPath);
+                return new List<GrindingSession>();
+            }
+                
+            var previousSessions =
+                Directory.EnumerateFiles($"{_historyPath}", "*.json")
```

Comment:
Can you elaborate about this `$"{_historyPath}"` ? 🤔

Reply:
currently each grindsession is saved down in Json/History folder

Reply:
I mean the syntax what being used :)